modules/machineset-yaml-azure.adoc

```diff
@@ -151,18 +143,17 @@ $  oc -n openshift-machine-api \
 ifndef::infra[]
 <2> Specify the node label to add.
 <3> Specify the infrastructure ID, node label, and region.
-<4> Optional: Specify the machine set name to enable the use of availability sets. This setting only applies to new compute machines.
-<5> Specify the region to place machines on.
-<6> Specify the zone within your region to place machines on. Be sure that your region supports the zone that you specify.
 endif::infra[]
 ifdef::infra[]
 <2> Specify the `<infra>` node label.
 <3> Specify the infrastructure ID, `<infra>` node label, and region.
+endif::infra[]
 <4> Optional: Specify the machine set name to enable the use of availability sets. This setting only applies to new compute machines.
-<5> Specify a taint to prevent user workloads from being scheduled on infra nodes.
+<5> Specify an image that is compatible with your instance type. If your instance type is only compatible with Hyper-V generation V2 images, you must use a Hyper-V generation V2 image and append `-gen2` to the `resourceID` value.
```

Comment:
I am happy with this and think it is clear enough. 

If we are telling users to edit existing machinesets, it is possible-even probable-the image id would be a gen2 image already. So perhaps we want to say something like:

```suggestion
<5> Specify an image that is compatible with your instance type. The Hyper-V generation V2 images created by the installer have a `-gen2` suffix, while V1 images have the same name without the suffix.
```

Reply:
Aha - that is very helpful, and I think will help users better than what I have here. Thanks!